### PR TITLE
version update 3.2.0

### PR DESCRIPTION
**Pull request by d-cunningham**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by d-cunningham** [comment]
@conda-forge-admin, please rerender

**Comment by d-cunningham** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -12,47 +12,40 @@ source:
   sha256: {{ sha256sum }}
 
 build:
-  number: 3
-  skip: True  # [win and py27]
-  detect_binary_files_with_prefix: False
+  number: 0
+  skip: True  # [win][py36]
```
Comment (dhirschfeld) [inline recipe/meta.yaml:16]: I'm not sure that syntax works? Did you mean:
```suggestion
  skip: True  # [win or py36]
```

Comment (d-cunningham) [reply recipe/meta.yaml]: Yes, that is what I meant. Thank you. Fixed it.

**Comment by dhirschfeld** [comment]
@conda-forge-admin, please rerender

**Comment by dhirschfeld** [comment]
Sorry, forgot to merge this one

**Pull request merged**